composer.json

```diff
@@ -40,7 +40,7 @@
         "beheh/flaps": "dev-master",
         "doctrine/cache": "^1.6.2",
         "ramsey/uuid": "^3.4.1",
-        "barryvdh/laravel-cors": "^0.9.2",
+        "barryvdh/laravel-cors": "^0.11.0",
```

Comment:
We should add
"asm89/stack-cors": "1.2.0" since we are dependent on it now separate from laravel-cors